package.json

```
@@ -28,8 +28,7 @@
     "app:storybook:serve": "nx storybook:serve mudita-center",
     "app:storybook:build": "nx storybook:build mudita-center",
     "e2e:test": "nx e2e:test mudita-center-e2e",
-    "e2e:test:headless": "nx e2e:test:headless mudita-center-e2e",
-    "e2e:test:linux": "xvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' npm run e2e:test:headless",
+    "e2e:test:linux": "xvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' nx e2e:test:headless mudita-center-e2e",
```

why e2e:test:headless? I recomend chnage npm script name to e2e:test:cicd

```suggestion
    "e2e:test:linux": "xvfb-run --auto-servernum --server-args='-screen 0 1024x768x24' nx e2e:test:cicd mudita-center-e2e",
```